src/com/LinkedList.java

```diff
@@ -0,0 +1,393 @@
+package com;
+import java.io.IOException;
+import java.util.Scanner;
+
+
+/**
+ * @author avisharma
+ *This Class is class for customized linked list
+ */
+public class LinkedList {
+
+	Node head;
+	
+	final static String os = System.getProperty("os.name");
+	
+	
```

Comment:
For complex methods like this rotatelist there should be online comments to help understand the logic